src/helpers/properties.ts

```diff
@@ -45,25 +66,33 @@ export class Properties {
         return this.api.GetLine(modelID, id, recursive, inverse);
     }
 
-    async getPropertySets(modelID: number, elementID: number, recursive = false) {
-        return await this.getRelatedProperties(modelID, elementID,recursive, 'IsDefinedBy', 'RelatingPropertyDefinition');
+    async getPropertySets(modelID: number, elementID = 0, recursive = false) {
```

Comment:
The 0 is used to have an easy way to get all propertysets or materials in an IfcModel.
As you said, there is no expressID 0, so it can be used as a 'fake' expressID for the whole project in `getRelatedProperties`.